### PR TITLE
Requirement update

### LINKED ISSUE #629: Wrong model  caching info and code in OpenVINO API sample
**Describe the bug**
OpenVINO API [notebook](https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/002-openvino-api/002-openvino-api.ipynb) contains incorrect information and code about the model caching, for example, "Model caching is not available on CPU devices."

 **Expected behavior**
Regardless of the device, caching works smoothly and there should be no special checks or comments about the supported devices.

### PR DESCRIPTION
**Pull request by AnishaUdayakumar**
Release version 2022.2.0 update in requirements and ignore notebooks 203 and 219 for the CI. Fixes #629

### CODE REVIEW

**Review by paularamo — APPROVED** [review]
@ryanloney Could you please take a look of this changes? We should come back to the jumped notebooks.

File: .github/workflows/nbval.yml [hunk 1]
```diff
@@ -188,7 +188,7 @@ jobs:
       if: runner.os == 'MacOS'
       run: |
         chmod +x .ci/validate_notebooks.sh
-        .ci/validate_notebooks.sh 107-speech-recognition-quantization 114-quantization-simplified-mode 115-async-api
+        .ci/validate_notebooks.sh 107-speech-recognition-quantization 114-quantization-simplified-mode 115-async-api 203-meter-reader 219-knowledge-graphs-conve
```
Comment (adrianboguszewski) [inline .github/workflows/nbval.yml:191]: You're ignoring these notebooks only for mac, not for all oses. Look at line 188

File: notebooks/002-openvino-api/002-openvino-api.ipynb [hunk 2]
```diff
@@ -126,32 +126,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from openvino.runtime import Core\n",
+    "from openvino.runtime import serialize, Core\n",
     "\n",
     "ie = Core()\n",
-    "onnx_model_path = \"model/segmentation.onnx\"\n",
-    "model_onnx = ie.read_model(model=onnx_model_path)\n",
-    "compiled_model_onnx = ie.compile_model(model=model_onnx, device_name=\"CPU\")"
```
Comment (adrianboguszewski) [inline notebooks/002-openvino-api/002-openvino-api.ipynb:134]: Why are these lines deleted?

File: notebooks/002-openvino-api/002-openvino-api.ipynb [hunk 3]
```diff
@@ -126,32 +126,11 @@
...
-   "execution_count": null,
-   "id": "a4fb397f",
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "from openvino.offline_transformations import serialize\n",
-    "\n",
-    "serialize(model=model_onnx, model_path=\"model/exported_onnx_model.xml\", weights_path=\"model/exported_onnx_model.bin\")"
+    "model = ie.read_model(\"model/segmentation.onnx\")\n",
+    "serialize(model, \"model/exported_onnx_model.xml\")"
```
Comment (adrianboguszewski) [inline notebooks/002-openvino-api/002-openvino-api.ipynb:133]: According to the contribution guide, all arguments should have corresponding keywords.

**Review by adrianboguszewski — CHANGES_REQUESTED** [review]

File: .github/workflows/nbval.yml (hunk 1 above)
Comment (AnishaUdayakumar) [reply .github/workflows/nbval.yml:191]: Thanks for spotting that. I will include it for OS: Windows and Linux as well.

File: notebooks/002-openvino-api/002-openvino-api.ipynb (hunk 2 above)
Comment (AnishaUdayakumar) [reply notebooks/002-openvino-api/002-openvino-api.ipynb:134]: 132 and 133 is rewritten to model = ie.read_model("model/segmentation.onnx")
I will include the step to show "How to compile the segmentation model "
Thanks for the feedback

File: notebooks/002-openvino-api/002-openvino-api.ipynb [hunk 4]
```diff
@@ -407,6 +410,19 @@
...
+   "id": "956f8cc4-a944-4470-8cc6-6be0ce038ea8",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "input_data = np.expand_dims(np.transpose(resized_image, (2, 0, 1)), 0).astype(np.float32)\n",
+    "input_data.shape"
+   ]
+  },
```
Comment (adrianboguszewski) [inline notebooks/002-openvino-api/002-openvino-api.ipynb:425]: This is to remove

File: notebooks/002-openvino-api/002-openvino-api.ipynb [hunk 5]
```diff
@@ -129,35 +129,38 @@
     "from openvino.runtime import Core\n",
     "\n",
     "ie = Core()\n",
-    "onnx_model_path = \"model/segmentation.onnx\"\n",
-    "model_onnx = ie.read_model(model=onnx_model_path)\n",
-    "compiled_model_onnx = ie.compile_model(model=model_onnx, device_name=\"CPU\")"
+    "segmentation_model_onnx = \"model/segmentation.onnx\"\n",
+    "\n",
+    "model = ie.read_model(model=segmentation_model_onnx)\n",
+    "compiled_model = ie.compile_model(model=model, device_name=\"CPU\")"
```
Comment (adrianboguszewski) [inline notebooks/002-openvino-api/002-openvino-api.ipynb:135]: This should be restored to the previous state

File: notebooks/002-openvino-api/002-openvino-api.ipynb [hunk 6]
```diff
@@ -129,35 +129,38 @@
...
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "06f4a187",
+   "id": "3d6a41d9-a2de-40a8-b925-fe3e817866d4",
    "metadata": {},
    "source": [
-    "The ONNX model can be exported to OpenVINO IR with `.serialize()`:"
+    "The ONNX model can be exported to OpenVINO IR with .serialize():"
```
Comment (adrianboguszewski) [inline notebooks/002-openvino-api/002-openvino-api.ipynb:143]:
```suggestion
    "The ONNX model can be exported to OpenVINO IR with `serialize():`"
```

File: notebooks/002-openvino-api/002-openvino-api.ipynb [hunk 7]
```diff
@@ -638,28 +653,25 @@
...
     "\n",
-    "from openvino.runtime import Core, PartialShape\n",
+    "from openvino.runtime import Core\n",
     "\n",
     "ie = Core()\n",
     "\n",
-    "device_name = \"GPU\"  # Model Caching is not available for CPU\n",
+    "device_name = \"GPU\" \n",
     "\n",
-    "if device_name in ie.available_devices and device_name != \"CPU\":\n",
```
Comment (adrianboguszewski) [inline notebooks/002-openvino-api/002-openvino-api.ipynb:647]: This if should be here as there could be a situation the user doesn't have GPU.

File: notebooks/002-openvino-api/002-openvino-api.ipynb [hunk 8]
```diff
@@ -677,12 +689,11 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "if device_name in ie.available_devices and device_name != \"CPU\":\n",
```
Comment (adrianboguszewski) [inline notebooks/002-openvino-api/002-openvino-api.ipynb:680]: The same here

**Review by adrianboguszewski — CHANGES_REQUESTED** [review]

File: notebooks/002-openvino-api/002-openvino-api.ipynb (hunk 7 above)
Comment (AnishaUdayakumar) [reply notebooks/002-openvino-api/002-openvino-api.ipynb:647]: Sure. I have included the if to cover the scenario.

File: notebooks/002-openvino-api/002-openvino-api.ipynb (hunk 8 above)
Comment (AnishaUdayakumar) [reply notebooks/002-openvino-api/002-openvino-api.ipynb:680]: updated the change with if

File: notebooks/002-openvino-api/002-openvino-api.ipynb (hunk 4 above)
Comment (AnishaUdayakumar) [reply notebooks/002-openvino-api/002-openvino-api.ipynb]: I have removed the redundant cell

File: notebooks/002-openvino-api/002-openvino-api.ipynb (hunk 5 above)
Comment (AnishaUdayakumar) [reply notebooks/002-openvino-api/002-openvino-api.ipynb]: Reverted to previous state

File: notebooks/002-openvino-api/002-openvino-api.ipynb [hunk 9]
```diff
@@ -130,34 +130,37 @@
     "\n",
     "ie = Core()\n",
     "onnx_model_path = \"model/segmentation.onnx\"\n",
+    "\n",
     "model_onnx = ie.read_model(model=onnx_model_path)\n",
-    "compiled_model_onnx = ie.compile_model(model=model_onnx, device_name=\"CPU\")"
+    "compiled_model = ie.compile_model(model=model_onnx, device_name=\"CPU\")"
```
Comment (adrianboguszewski) [inline notebooks/002-openvino-api/002-openvino-api.ipynb:135]:
```suggestion
    "compiled_model_onnx = ie.compile_model(model=model_onnx, device_name=\"CPU\")"
```
